pyisic/_standards/ateco/ateco.py

```diff
@@ -0,0 +1,6108 @@
+"""`ATECO Standard <https://www.istat.it/classificazione/documenti-ateco/>`_."""
```

Comment:
Curios - which dataset did you use? Could we not have these be English using [Struttura ATECO 2025 italiano inglese](https://www.istat.it/wp-content/uploads/2024/12/StrutturaATECO-2025-IT-EN-1.xlsx)?

Reply:
This one: https://www.istat.it/wp-content/uploads/2024/12/Struttura-ATECO-2025-vs-NACE.xlsx, pushed the english description!

Reply:
I used the "TITOLO_INGLESE_ATECO_2025" column.